api/src/main/java/org/pmiops/workbench/BlockscoreServiceImpl.java

```diff
@@ -20,10 +23,44 @@ public BlockscoreServiceImpl(Provider<CloudStorageService> cloudStorageServicePr
   }
 
   public PaginatedResult<Person> listPeople() {
+    return getClient().listPeople();
+  }
+
+  public Address createAddress(
```

Comment:
This method doesn't hurt anything, but doesn't seem totally necessary... couldn't the client just as easily do this?

It also seems misleading to have "createAddress" which constructs a POJO and returns it vs. "createPerson" which constructs a POJO and then actually calls the API to create it in BlockScore... so if we're going to keep this method, rename it?

Reply:
Yeah, my original goal was to prevent users of this class from having to import BlockScore's models, but that fell through so I agree this doesn't add value.